common/enablers/enableRoundsHandler.go

```diff
@@ -0,0 +1,52 @@
+package enablers
+
+import (
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-go-core/core/atomic"
+	"github.com/ElrondNetwork/elrond-go/config"
+)
+
+// TODO uncomment all from this file and from the test file and replace Example with something else
+// const exampleName = "Example"
```

Comment:
[comment]

A bit tricky that we have to keep consistency between this constants and the ones from config.

Reply:
these are safe to use. If we mangled them, the node will not start